configs/config_lock_redis_cluster.json

```diff
@@ -0,0 +1,76 @@
+{
+  "servers": [
+    {
+      "default_log_path": "stdout",
+      "default_log_level": "DEBUG",
+      "routers": [
+        {
+          "router_config_name": "actuator_dont_need_router"
+        }
+      ],
+      "listeners": [
+        {
+          "name": "grpc",
+          "address": "127.0.0.1:34904",
+          "bind_port": true,
+          "filter_chains": [
+            {
+              "filters": [
+                {
+                  "type": "grpc",
+                  "config": {
+                    "server_name": "runtime",
+                    "grpc_config": {
+                      "hellos": {
+                        "helloworld": {
+                          "hello": "greeting"
+                        }
+                      },
+                      "lock": {
+                        "redis_cluster": {
+                          "metadata": {
+                            "concurrency": "0",
```

Comment:
Hi,could you please add a component doc that explains what configuration items this component supports?
You can take [the doc of standalone redis lock](https://mosn.io/layotto/#/zh/component_specs/lock/redis) as an example

Reply:
ok,thanks for your advice

Reply:
hi, I have added the doc of config_lock_redis_cluster.json in https://github.com/whalesongAndLittleFish/layotto/blob/add_red_lock/docs/zh/component_specs/lock/redis.md